### PR TITLE
step lr_scheduler on epoch end in hapi/model.py fit

### PR DESCRIPTION
**Pull request by heavengate**
### PR types
Function optimization

### PR changes
Others

### Describe
`LRScheduler` in `paddle.optimizer.lr_scheduler` decay learning rate by epoch and `LRScheduler.step()` should be called on each epoch end, add `LRScheduler.step()` in `fit` in `hapi/model.py`

Tested on static/dygraph TSM/YOLOv3 ok

### CODE REVIEW

**Review by saxon-zh — DISMISSED** [review]
LGTM

**Review by LielinJiang — DISMISSED** [review]

**Review by qingqing01 — APPROVED** [review]
